test/test_scenario_1.py

```diff
@@ -46,15 +42,6 @@ def test_tropo_delay_ERA5T(tmp_path):
     core_test_tropo_delay(tmp_path, modelName="ERA5T")
 
 
-@pytest.mark.long
```

Comment:
@the-garlic-os is there a reason for deleting this particular test and not others?

Reply:
This test was actually a duplicate of the one that was on line 77. `test_tropo_delay_GMAO` still exists and still runs in the test suite.